.github/workflows/main.yml

```diff
@@ -65,3 +65,6 @@ jobs:
 
     - name: Test with pytest
       run: ./manage.py test
+
+    - name: Check types
+      run: mypy bread/forms
```

Comment:
Wait, is there a confusion between ```bread/forms``` and ```bread/layout/components/forms``` maybe?

Reply:
Not really. These are the typehints that were necessary for `mypy bread/forms` to succeed...